examples/integrations/oci/vault-cdi/src/test/resources/META-INF/microprofile-config.properties

```diff
@@ -13,5 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-Args=\
---initialize-at-run-time=com.oracle.bmc.auth.AbstractFederationClientAuthenticationDetailsProviderBuilder$SessionKeySupplierImpl
+
+oci.properties.vault-ocid=a
```

Comment:
Is this intended? Is there appropriate explanation in example readme that it must be filled in?

Reply:
This test was just starting server and not doing anything so removed it.